pkg/manifests/manifests.go

```diff
@@ -28,6 +29,9 @@ const (
 	nodeResolverScriptAsset         = "assets/node-resolver/update-node-resolver.sh"
 	nodeResolverServiceAccountAsset = "assets/node-resolver/service-account.yaml"
 
+	DNSNameResolverTechPreviewNoUpgradeCRDAsset = "assets/dnsnameresolver/0000_70_dnsnameresolver_00-techpreview.crd.yaml"
+	DNSNameResolverCustomNoUpgradeCRDAsset      = "assets/dnsnameresolver/0000_70_dnsnameresolver_00-customnoupgrade.crd.yaml"
+
```

Comment:
It's not clear to me why you need both a tech preview and customnoupgrade CRD in the same release?